### PR TITLE
Assignment 2b David Cristóbal

### PR DESCRIPTION
**Pull request by juanked**

### CODE REVIEW

File: Assignment2/juanked-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+    "@context": {
+        "vocab": "http://upmsemanticweb.com/students#",
+        "fullName": "vocab:fullName",
+        "hometown": "vocab:hometown",
+        "age": "vocab:age",
+        "almaMater": "vocab:almaMater",
+        "familiarsNames": "vocab:familiarsNames",
+        "father": "familiarsNames:father",
+        "mother": "familiarsNames:mother",
```
Comment (paoespinozarias) [inline Assignment2/juanked-fromJSONtoJSON-LD.jsonld:10]: These attribute definitions work, but semantically they aren't ok. The best way is to use a prefix and add the name of the concept/property you want to use e.g. "mother":"yourvocab:mother".
You don't need to change them, but remember to provide good definitions for the future.

File: Assignment2/juanked-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+    "@context": {
+        "obj": "http://www.w3.org/2001/obj",
```
Comment (paoespinozarias) [inline Assignment2/juanked-fromRDFtoJSON-LD.jsonld:3]: Add / at the end of the URI

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by paoespinozarias** [comment]
The changes requested on Assignment 2b are still pending. In the meantime, @ocorcho can review Assignment 3.

**Comment by ocorcho** [comment]
Assignment 3 is ok

File: Assignment2/juanked-fromRDFtoJSON-LD.jsonld (hunk 2 above)
Comment (juanked) [reply Assignment2/juanked-fromRDFtoJSON-LD.jsonld]: Hi, I did add the / at the end of the URI (as you can see in my "change done" commit), I don't know what else should change to satisfy the change request

Comment (paoespinozarias) [reply Assignment2/juanked-fromRDFtoJSON-LD.jsonld]: Sorry, you are right. It is ok.

**Review by paoespinozarias — APPROVED** [review]

**Pull request merged**